esp/src/eclwatch/WsDFUXref.js

```diff
@@ -0,0 +1,52 @@
+/*##############################################################################
+#    HPCC SYSTEMS software Copyright (C) 2012 HPCC Systems®.
```

Comment:
Actually I believe we don't need this copyright message in each file any more as the single licence.txt should suffice?